js/plain-text-evaluator/PlainTextEvaluator.js

```diff
@@ -133,6 +137,10 @@ class PlainTextEvaluator extends Evaluator {
     return '\n';
   }
 
+  getWordbreak() {
+    return '\n';
```

Comment:
I do not think this is the correct return.

The behavior we are implementing is HTML-specific (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) where "the browser may optionally break a line". If we return `\n` here, we are forcing a line break, which I feel is incorrect. If the end user wanted to guarantee a line break they'd use Newline instead of WorkBreak. 

The end user is intended for maybe a line break, deferring to the consuming renderer to decide if there's a linebreak. I think we need to return empty string and allow the code consuming the result to figure out where to make the line break, if at all. That would best match up to the HTML spec IMO.